src/nitypes/time/_conversion.py

```diff
@@ -8,13 +8,14 @@
 import hightime as ht
 from typing_extensions import TypeAlias
 
+import nitypes.bintime as bt
 from nitypes._exceptions import invalid_arg_type, invalid_requested_type
 
-_AnyDateTime: TypeAlias = Union[dt.datetime, ht.datetime]
-_TDateTime = TypeVar("_TDateTime", dt.datetime, ht.datetime)
+_AnyDateTime: TypeAlias = Union[bt.AbsoluteTime, dt.datetime, ht.datetime]
```

Comment:
Open question on naming: do we want to use `nitypes.bintime.datetime` and `nitypes.bintime.timedelta`, or do we want to use PascalCase as I am currently doing?

If PascalCase:
- Should the date time be called DateTime or AbsoluteTime?
- Should the time delta be called TimeDelta, TimeValue, or TimeInterval?

Reply:
https://peps.python.org/pep-0008/#class-names

> Class names should normally use the CapWords convention.
>
> The naming convention for functions may be used instead in cases where the interface is documented and used primarily as a callable.
> 
> Note that there is a separate convention for builtin names: most builtin names are single words (or two words run together), with the CapWords convention used only for exception names and builtin constants.

I'll rename to DateTime and TimeDelta.